File: Java/Java source-code/mean_absolute_error.java


public class mean_absolute_error {
    public static double mean_absolute_error(Integer[] a, Integer[] b) {
        int sum = 0;
        for (int i = 0; i < a.length; i++) {
            sum += Math.abs(a[i] - b[i]);
        }
        return (double) sum / a.length;
    }
}

File: Methods/Methods-C++/mean_absolute_error.cpp
#include <iostream>

double mean_absolute_error(int a[], int b[], int n) {
    int sum = 0;
    for (int i = 0; i < n; i++) {
        sum += abs(a[i] - b[i]);
    }
    return (double) sum / n;
}
